.github/workflows/test.yml

```diff
@@ -68,6 +68,32 @@ jobs:
           cd omniscidb
           conda run -n ${{ env.CONDA_ENV }} bash scripts/conda/test.sh
 
+      - name: Run ASAN test
+        if: inputs.test == 'asan'
+        env:
+          ASAN_OPTIONS: verbosity=1
+
+        run: |
+          sudo sh -c 'echo 2 >/proc/sys/vm/overcommit_memory'
+          cd omniscidb
+
+          cd build/Tests # build cache
+          conda run -n ${{ env.CONDA_ENV }} ./ArrowBasedExecuteTest --build-rel-alg-cache=cache.txt
+
+          cd ../.. # rebuild with ASAN
+          conda run -n ${{ env.CONDA_ENV }} cmake -B ./build -S . -DENABLE_TESTS=on -DENABLE_CUDA=off -DENABLE_ASAN=on
```

Comment:
Is this for saving up space or?

Reply:
we need to run an unmodified test first to cache calcite queries and responses, this is done in this patch
then we need to recompile the whole project with ASAN and run the test with cached queries/responses, this is not fully finished

Reply:
Ah, I see